x-pack/plugins/ingest_manager/public/applications/ingest_manager/types/intra_app_route_state.ts

```diff
@@ -29,9 +29,18 @@ export interface AgentConfigDetailsDeployAgentAction {
   onDoneNavigateTo?: Parameters<ApplicationStart['navigateToApp']>;
 }
 
+/**
+ * Supported routing state for the agent config details page routes with deploy agents action
```

Comment:
So is this file where all of the round trips are registered basically?

Reply:
this is where you define the types for which actions a page will support

Reply:
noice